prime-router/metadata/schemas/Experity/experity-covid-19.schema

```diff
@@ -0,0 +1,644 @@
+      csvFields: [{name: Testing_lab_accession_number}]
+
+#demographics
+    - name: patient_id
+      csvFields: [{name: Patient_ID}]
+
+    - name: patient_last_name
+      csvFields: [{name: Patient_last_name}]
+
+    - name: patient_middle_name
+      csvFields: [{name: Patient_middle_name}]
+
+    - name: patient_first_name
+      csvFields: [{name: Patient_first_name}]
+
+    - name: patient_street
+      csvFields: [{name: Patient_street}]
+
+    - name: patient_street2
+      csvFields: [{name: Patient_street2}]
+
+    - name: patient_city
+      csvFields: [{name: Patient_city}]
+
+    - name: patient_state
+      csvFields: [{name: Patient_state}]
+
+    - name: patient_zip_code
+      csvFields: [{name: Patient_zip_code}]
+
+    - name: patient_county
+      type: TABLE
+      table: zip-code-data
+      tableColumn: county
+      mapper: zipCodeToCounty(patient_zip_code)
+
+    - name: patient_phone_number
+      csvFields: [{name: Patient_phone_number}]
+
+    - name: patient_dob
+      csvFields: [{name: Patient_DOB, format: "M/d/yyyy"}]
+
+    - name: patient_gender
+      type: CODE
+      valueSet: sender-automation/gender
+      documentation: Translate multiple inbound Gender values to RS values
+      csvFields: [{ name: Patient_gender, format: $display}]
+      default: U
+
+    - name: patient_ethnicity
+      type: CODE
+      valueSet: sender-automation/ethnicity
+      documentation: Translate multiple inbound ethnicity values to RS / OMB values
+      csvFields: [{ name: Patient_ethnicity, format: $display}]
+      default: U
+
+    - name: patient_race
+      type: CODE
+      valueSet: sender-automation/race
+      documentation: Translate multiple inbound Race values to RS / OMB values
+      csvFields: [{ name: Patient_race, format: $display}]
+      default: UNK
+
+# Specimen/Order Info
+    - name: specimen_collection_date_time
+      csvFields: [{name: Specimen_collection_date_time, format: "M/d/yyyy H:nn"}]
+
+    - name: order_test_date
+      csvFields: [{name: Order_test_date, format: "M/d/yyyy"}]
+
+    - name: testing_lab_specimen_received_datetime
+      csvFields: [{name: Specimen_collection_date_time, format: "M/d/yyyy H:nn"}]
+
+    - name: test_result_date
+      csvFields: [{name: Date_result_released, format: "M/d/yyyy H:nn"}]
+
+    - name: test_result_report_date
+      csvFields: [{name: Test_date, format: "M/d/yyyy"}]
+
+    - name: test_result
+      type: CODE
+      valueSet: sender-automation/test_result
+      documentation: Translate multiple inbound Test Result values to RS values
+      csvFields: [{ name: Test_result_coded, format: $display}]
+
+    - name: test_result_status
+      csvFields: [{name: Test_result_status}]
+      default: F
+
+    - name: filler_order_id
+      csvFields: [{name: Testing_lab_accession_number}]
+
+
+    - name: specimen_type
+      type: CODE
+      csvFields: [{name: Specimen_type_description, format: $alt}]
+      default: "258500001"
+      altValues:
```

Comment:
How do we know these map properly to the internally expected values, and someone didn't make a typo?   This is a big weakness in our "alt" values feature.

Does state of Louisiana get to approve this data before "GUC" goes live?   If not, let's chat, please.   The Receivers on boarding team has asked that the state validate data before going live with new senders.

Reply:
1) I didn't attach the files to this Issue, but I've saved my last test CSV and the resulting HL7 output that I used to validate these fields.  Note, I didn't attach them because they are large, and even though they are de-identified, they still contain the practice and provider names, addresses and phone #s.  If you want someone else to review, I have the files saved to our GDrive. 

2) We send production data to the States production environment, so they can validation the results before letting them process into their production system.  The process is: 1) we contact them with the CLIA#s they will receive; 2) They setup their Production receiver process to block the results for these CLIA#s, so they can validate them first; 3)  Once the State approves, they modify their side to allow processing into Production, and we then send all results.